src/modules/ekf2/EKF/mag_3d_control.cpp

```diff
@@ -65,8 +64,6 @@ void Ekf::controlMag3DFusion(const magSample &mag_sample, const bool common_star
 				       && _control_status.flags.mag_aligned_in_flight
```

Comment:
TODO (later): Could we do something to safely allow full mag_3D before mag_aligned_in_flight (eg ground testing, etc)?

Reply:
Yes, good point. If this work we could get rid of the separate mag bias estimator used for quick calibration and all the complicated logic that merges both results.